README.md

```diff
@@ -87,7 +87,7 @@ Assignments by default are assumed to be inside `db/assignments`. You can change
 
 Each assignment should extend the `BaseJudge` class or the `SolidityJudge` class in the case of the smart contract. The assignments are parameterized for each user and each user has a public and a private aux. Both the public and the private aux are created once (See `db/models/parameterizedassignment.js`).
 
-Each assignment has to implement `judge (aux, user, assignment, solution)` and `aux (user, assignment)` functions. `judge` function takes as input an `aux`, a `user`, an `assignment` and a `solution`, and returns `1` in case of success and `0` otherwise. The `aux` function takes as input the `user` and the `assignment` and return an aux `{ private: 'privateAux', public: 'publicAux' }` where `private` and `public` can be of any type and can be omitted.
+Each assignment has to implement `judge (aux, user, assignment, solution)` and `aux (user, assignment)` functions. `judge` function takes as input an `aux`, a `user`, an `assignment` and a `solution`, and returns and object containing the grade and a message to show to the user. The return object's format must be: `{ grade: 0, msg: 'A message' }` The `aux` function takes as input the `user` and the `assignment` and return an aux `{ private: 'privateAux', public: 'publicAux' }` where `private` and `public` can be of any type and can be omitted.
```

Comment:
returns and object -> returns an object
and return an aux -> and returns an aux
`{ private: 'privateAux', public: 'publicAux' }` -> `{ private: someObj, public: anotherObj }` (don't use strings)

Reply:
Thanks!